f/medical/fn_medicalPreInit.sqf

```diff
@@ -0,0 +1,60 @@
+// F3 - Medical Systems Pre Initialisation
+// Credits: Please see the F3 online manual (http://www.ferstaberinde.com/f3/en/)
+// ====================================================================================
+
+// Wait until medical param is initialized by the pre init param processor
+waitUntil
+{
+	!(isNil {missionNamespace getVariable "f_var_medical"})
+};
```

Comment:
Pre-Init is an unscheduled environment, this might need integration with @Sniperhid`s new param preprocessor.